airbyte-integrations/connectors/source-sftp-bulk/integration_tests/utils.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+import json
+import logging
+import os
+import re
+from typing import Any, Mapping, Union
+
+logger = logging.getLogger("airbyte")
+
+PRIVATE_KEY = str
```

Comment:
Shouldn't an empty string be used here as a default value?

Reply:
agreed